lib/cocoapods-downloader/gem_version.rb

```diff
@@ -3,6 +3,6 @@ module Downloader
     # @return [String] Downloader’s version, following
     #         [semver](http://semver.org).
     #
-    VERSION = '1.2.2'.freeze
+    VERSION = '1.2.3'.freeze
```

Comment:
Could you remove the version bump? That will be handled upon release

Reply:
+1

Reply:
Done